crates/next-core/js/src/overlay/internal/container/Errors.tsx

```diff
@@ -166,41 +166,58 @@ function useDisplayState(
 }
 
 const enum TabId {
-  TurbopackIssues = "turbopack-issues",
+  TurbopackErrors = "turbopack-issues",
+  TurbopackWarnings = "turbopack-warnings",
   RuntimeErrors = "runtime-errors",
 }
 
+const TAB_PRIORITY = [
+  TabId.TurbopackErrors,
+  TabId.RuntimeErrors,
+  TabId.TurbopackWarnings,
+];
+
 export function Errors({ issues, errors }: ErrorsProps) {
   const [readyErrors, isLoading] = useResolvedErrors(errors);
 
-  const hasIssues = issues.length !== 0;
-  const hasIssueWithError = issues.some((issue) =>
+  const turbopackWarnings = issues.filter(
+    (issue) => !["bug", "fatal", "error"].includes(issue.severity)
+  );
+  const turbopackErrors = issues.filter((issue) =>
     ["bug", "fatal", "error"].includes(issue.severity)
   );
 
+  const hasTurbopackWarnings = turbopackWarnings.length > 0;
+  const hasTurbopackErrors = turbopackErrors.length > 0;
+
   const hasErrors = errors.length !== 0;
   const hasServerError = readyErrors.some((err) =>
     ["server", "edge-server"].includes(getErrorSource(err.error) || "")
   );
 
   // TODO for now it's already closable, but in future we might want to block users from using a broken app
-  // const isClosable = !isLoading && !hasIssueWithError && !hasServerError;
+  // const isClosable = !isLoading && !hasTurbopackErrors && !hasServerError;
   const isClosable = true;
 
   const defaultTab =
-    hasIssueWithError || !hasErrors
-      ? TabId.TurbopackIssues
-      : TabId.RuntimeErrors;
+    TAB_PRIORITY.filter(
```

Comment:
[Array.prototype.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) might be more suitable

Reply:
true, changed it